packages/gitlab-backend/src/plugin.ts

```diff
@@ -23,6 +24,27 @@ export const catalogPluginGitlabFillerProcessorModule = createBackendModule({
     },
 });
 
+export const catalogPluginGitlabProjectIdProcessorModule = createBackendModule({
```

Comment:
Because there is a variable to activate/deactivate, we don't need to create a new backend module; we can use the other [one](https://github.com/immobiliare/backstage-plugin-gitlab/blob/2dbaf5fa5b9e9c88445980d967f93763681bd592/packages/gitlab-backend/src/plugin.ts#L10). In this way, we force even the processor's ordering.